### ISSUE TITLE
[Feature Request] Support for Laser Point Cloud (las / laz file) Import / Processing

### ISSUE THREAD

**Issue by digital-idiot** [domlysz/BlenderGIS#256]
Given the 3D processing and visualization capabilities of Blender, I think it would be very useful if georeferenced Point Cloud from LiDAR scans can be imported into blender.

**Comment by domlysz** [comment]
yes, wanted to work on this years ago but I'm running out of time.

**Comment by digital-idiot** [comment]
@domlysz I can help with the very limited coding skill I have.

**Comment by domlysz** [comment]
#247
